src/Form/FormAnalyzer.js

```diff
@@ -49,20 +56,37 @@ class FormAnalyzer {
         return this
     }
 
+    /**
+     *
+     * @param {object} p
```

Comment:
It's short for `parameter` but the name is not used anywhere because it's destructured in the function signature. It's just needed to write the jsdoc this way. I could have used a different jsdoc style but the inline descriptions would be messy.